UPGRADE.md

```diff
@@ -2,6 +2,10 @@
 
 ## dev-develop
 
+### XML-Templates
+Blocks now supports `minOccurs="0"` and `maxOccurs > 256`. For that the validation was improved and for both negative
```

Comment:
yes if you remove the line it is unlimited! but before you cannot set a value bigger than 256 because type byte was used to validate!